lib/python/rose/stem.py

```diff
@@ -359,11 +360,24 @@ def _check_suite_version(self, fname):
         if not suite_rose_stem_version == ROSE_STEM_VERSION:
             raise RoseStemVersionException(suite_rose_stem_version)
 
+    def _get_localhost(self):
+        """Return the local host on which the rose-stem command was invoked."""
+        return socket.gethostname()
```

Comment:
You may want to use the `HostSelector.get_local_host` method in `rose.host_select` instead.